src/Models/ProductModel.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once 'src/Entities/Product.php';
+
+class ProductModel
+{
+    public static function getProductByCatId(int $cat_id, PDO $db): array
```

Comment:
There is no reason to abbreviate the word 'Category', you have tab-complete in your IDE.

Abbreviating to `$cat_id` here (and other places) is inconsistent with all the times elsewhere that the variable name is `$category_id` (because that is what comes out of the database).

Everywhere you have shortened 'category' to 'cat' please use the full word.